app/config/settings.py

```diff
@@ -1114,6 +1115,10 @@
 }
 # Number of minutes grace period before the container is stopped
 WORKSTATIONS_GRACE_MINUTES = 5
+# Feature Flag for reader study admin view
+READER_STUDY_VIEW_AS_USER_FEATURE = os.environ.get(
+    "READER_STUDY_VIEW_AS_USER_FEATURE", "False"
+)
```

Comment:
This won't give you the intended behaviour. In this case the default will be `READER_STUDY_VIEW_AS_USER_FEATURE == "False"`, so when you do `{% if reader_study_view_as_user_feature %}` in your template that will resolve to `True` as `bool("False") == True`. You need to use `strtobool`, see the other boolean settings.